clash-lib/src/Clash/Driver/Manifest.hs

```diff
@@ -161,6 +168,8 @@ instance ToJSON Manifest where
             ]
           )
         | (domNm, VDomainConfiguration{..}) <- HashMap.toList domains ]
+      , "dependencies" .= Aeson.object
+        [ "transitive" .= transitiveDependencies ]
```

Comment:
Does this need to be an object with a single key? Are you planning on adding other types of dependencies later?

Reply:
I intended to add only direct dependencies (which I think is neater), but only the transitive closure was easily obtainable. At some point I do want to add the former though. Given that the JSON is probably going to be parsed by Python's `json.load`, it'd be nice if it didn't break on adding it.